input/fsh/SD_Procedures.fsh

```diff
@@ -1,79 +1,155 @@
-* code from CancerRelatedSurgicalProcedureVS (extensible)
+    Invariant: teleradiotherapy-procedure-code-invariant
+    Description: "If the code 'Other Teleradiotherapy Modality, specify' is used, a second code from outside the original value set must be present."
+    Expression: "coding.where(code = 'OtherTeleradiotherapyModality').exists() implies coding.where(code != 'OtherTeleradiotherapyModality' and $this.memberOf('http://hl7.org/fhir/us/mcode/ValueSet/mcode-teleradiotherapy-modality-vs').not()).exists()"
+    Severity:   #error
 
-/*
-Extension: RadiationDose
-Id: mcode-radiation-dose
-Title:  "Radiation Dose"
-Description: "Information related to the dose of radiation received in a RadiationProcedure, including the dose per fraction, the number of fractions delivered, and the total radiation dose delivered."
-* value[x] 0..0
-* extension contains
-    RadiationDosePerFraction 0..1 and
-    RadiationFractionsDelivered 0..1 and
-    TotalRadiationDoseDelivered 0..1
+Profile:  BrachytherapyPrescriptionDelivery
+Parent:   USCoreProcedure
+Id:       brachytherapy-prescription-delivery
+Title:    "Brachytherapy Prescription Delivery"
+Description: "A summary of delivered brachytherapy treatment. The scope is a prescription consisting of one or multiple fractions. A new prescription delivery begins when there is a change in the target volume of a body site, treatment fraction size, modality, or treatment technique.Conformance statement: TBD"
+* insert RadiotherapyPrescriptionDeliveryRS
+* code from BrachytherapyModalityVS (required)
+* code obeys brachytherapy-code-invariant
+* code ^short = "Brachytherapy Modality"
+* code ^definition = "The modality of the brachytherapy procedure."
+* extension[radiotherapyTechnique].valueString from BrachytherapyTechniqueVS (extensible)
+* usedCode from BrachytherapyDeviceVS (extensible)
+
+    Invariant: brachytherapy-code-invariant
+    Description: "If the code representing 'Other brachytherapy, specify' is used, a second code from outside the original value set must be present."
+    Expression: "coding.where(code = 'OtherBrachytherapyModality').exists() implies coding.where(code != 'OtherBrachytherapyModality' and $this.memberOf('http://hl7.org/fhir/us/mcode/ValueSet/brachytherapy-modality-vs').not()).exists()"
+    Severity:   #error
+
+// TO DO: Use standard extension procedure-method IF the cardinality will be 0..1
+Extension: RadiotherapyTechnique
+Id: radiotherapy-technique
+Title: "Radiation Procedure Technique"
+Description: "The method by which a radiation modality is applied (e.g., intensity modulated radiation therapy, intraoperative radiation therapy)."
+* . ^short = "Radiation Procedure Technique"
+* . ^definition = "The method by which a radiation modality is applied (e.g., intensity modulated radiation therapy, intraoperative radiation therapy)."
+* ^context[0].type = #element
+* ^context[0].expression = "Procedure"
+* value[x] only string
```

Comment:
Why is this a `string` and not a `CodeableConcept`?

Reply:
Ah, that's because ASTRO doesn't want to go through a slow terminology process. The field moves too fast. 
Between us, there isn't any real difference between a self-created code system and string enumerations.